flatnav/util/SimdBaseTypes.h

```diff
@@ -75,27 +75,27 @@ struct simd4float32 : public simd128bit {
   explicit simd4float32(float x0, float x1, float x2, float x3)
       : simd128bit(_mm_setr_ps(x3, x2, x1, x0)) {}
 
-  simd4float32 operator+(const simd4float32 &other) const {
+  inline simd4float32 operator+(const simd4float32 &other) const {
     __m128 result = _mm_add_ps(_float, other._float);
     return simd4float32(result);
   }
 
-  simd4float32 operator*(const simd4float32 &other) const {
+  inline simd4float32 operator*(const simd4float32 &other) const {
     __m128 result = _mm_mul_ps(_float, other._float);
     return simd4float32(result);
   }
 
-  simd4float32 operator/(const simd4float32 &other) const {
+  inline simd4float32 operator/(const simd4float32 &other) const {
```

Comment:
It turns out inlining here is very helpful since these operations are called quite often.